src/languages/en.ts

```diff
@@ -2243,39 +2243,39 @@ export default {
                 },
             },
             reimbursableExpenses: {
-                label: 'Export reimbursable expenses as',
-                description: 'Reimbursable expenses will export as expense reports to Sage Intacct. Bills will export as vendor bills.',
+                label: 'Export out-of-pocket expenses as',
+                description: 'Set how out-of-pocket expenses export to Sage Intacct.',
                 values: {
                     [CONST.SAGE_INTACCT_REIMBURSABLE_EXPENSE_TYPE.EXPENSE_REPORT]: 'Expense reports',
                     [CONST.SAGE_INTACCT_REIMBURSABLE_EXPENSE_TYPE.VENDOR_BILL]: 'Vendor bills',
                 },
             },
             nonReimbursableExpenses: {
-                label: 'Export non-reimbursable expenses as',
+                label: 'Export company cards as',
                 description:
-                    'Non-reimbursable expenses will export to Sage Intacct as either credit card transactions or vendor bills and credit the account selected below. Learn more about assigning cards to individual accounts.',
+                    'Set how company card purchases export to Sage Intacct.',
```

Comment:
May be the previous text was more descriptive and helpful? 🤔

Reply:
The most helpful information can already be seen in the product - the text was just restating that, so it's pretty unnecessary!

Reply:
Okay, thanks for the context 👍